projects/kit/directives/skeleton/skeleton.style.less

```diff
@@ -17,16 +17,22 @@
         box-shadow: none !important;
         filter: contrast(0) brightness(0);
         animation: tuiSkeleton ease-in-out 1s infinite alternate;
+        user-select: none;
+        pointer-events: none;
 
         &[data-tui-skeleton] {
             background: transparent !important;
+            user-select: none;
```

Comment:
I don't think we need it here, it's the same element that already has those rules. Also, I believe, they are inherited by `::before` so it's not needed there too.

Reply:
I was thinking the same about this tag. Let me verify the `::before thing`

Reply:
Updated! Please review again @waterplea 
(Force-pushed again to correct scope for commitlint)